src/ripple/basics/Buffer.h

```diff
@@ -165,53 +173,59 @@ class Buffer
     void
     clear() noexcept
     {
-        p_.reset();
+        delete[] p_;
```

Comment:
Need to set `p_ = nullptr` here or we can double delete if `clear` is called twice.

Reply:
Oops!